File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1049.lua
a = io.read()
b = io.read()
c = io.read()

if(a=="vertebrado") then
    if(b=="ave") then
        if(c=="carnivoro") then
            print("aguia");
         else
            print("pomba");
         end
     else
        if(c=="onivoro") then
            print("homem");
        else
            print("vaca");
        end
    end
 else
    if(b=="inseto") then
        if(c=="hematofago") then
            print("pulga");
        else
            print("lagarta");
        end
    else
        if(c=="hematofago") then
            print("sanguessuga");
        else
            print("minhoca");
        end
    end
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1049.cpp
#include <iostream>

using namespace std;

int main(){
    string a, b, c;
    cin >> a >> b >> c;

    if(a=="vertebrado"){
        if(b=="ave"){
            if(c=="carnivoro"){
                cout << "aguia";
            } else{
                cout << "pomba";
            }
        } else{
            if(c=="onivoro"){
                cout << "homem";
            } else{
                cout << "vaca";
            }
        }
    } else{
        if(b=="inseto"){
            if(c=="hematofago"){
                cout << "pulga";
            } else{
                cout << "lagarta";
            }
        } else{
            if(c=="hematofago"){
                cout << "sanguessuga";
            } else{
                cout << "minhoca";
            }
        }
    }
    cout << endl;
}